### PR TITLE
Promote current pagerduty to prod

### PR DESCRIPTION
**Pull request by wshearn**
@jharrington22

### CODE REVIEW

**Comment by jharrington22** [comment]
LGTM @jfchevrette @mmclanerh if I could get a +1 from either of you also please. 

cc/ @jmelis

**Review by dgoodwin — APPROVED** [review]

**Review by jharrington22 — APPROVED** [review]
:+1:

**Comment by jharrington22** [comment]
Merging!

**Pull request merged**
